Sources/TestSupport/FileSystemExtensions.swift

```diff
@@ -48,3 +48,15 @@ extension InMemoryFileSystem {
         }
     }
 }
+
+public extension FileSystem {
+    /// Write to a file from a stream producer.
+    //
+    // FIXME: This is copy-paste from Commands/init.swift, maybe it is reasonable to lift it to Basic?
```

Comment:
I agree, being able to write file system contents is definitely general enough for FileSystem itself.